SAGE Journals.js

```diff
@@ -48,10 +49,10 @@ function detectWeb(doc, url) {
 function getSearchResults(doc, checkOnly) {
 	var items = {};
 	var found = false;
-	let rows = ZU.xpath(doc, '//div[contains(@class, "art_title")]/a[contains(@href, "/doi/full/10.") or contains(@href, "/doi/abs/10.") or contains(@href, "/doi/pdf/10.")][1]');
+	var rows = ZU.xpath(doc, '//span[contains(@class, "art_title")]/a[contains(@href, "/doi/full/10.") or contains(@href, "/doi/abs/10.") or contains(@href, "/doi/pdf/10.")][1] | //a[contains(concat( " ", @class, " " ), concat( " ", "ref", " " )) and contains(concat( " ", @class, " " ), concat( " ", "nowrap", " " ))] | //*[contains(concat( " ", @class, " " ), concat( " ", "hlFld-Title", " " ))]');
```

Comment:
By convention we use `let` where possible.

Reply:
With this Xpath expression the originally present  `<div>` case would no longer be detected.

Reply:
> By convention we use `let` where possible.

ok.

> With this Xpath expression the originally present `<div>` case would no longer be detected.

we already tested successfully with `span`

Reply:
There there criminological contexts in which `div` is needed (cf. a850eba3023e80ae5ac3fdfc86cb9421965d7038)